pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/rancher/norman/objectclient"
 	"github.com/rancher/norman/types/convert"
 	wranglerv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/controllers/management/auth/roletemplates"
```

Comment:
New code is actually located at `pkg/controllers/managementuser/rbac/handler_base.go` not `management/auth`. Which one is incorrect, this import or the location of new files?